src/app/wallet/views/asset_list.nim

```diff
@@ -8,6 +8,7 @@ type
     Value = UserRole + 3,
     FiatValue = UserRole + 4
     Address = UserRole + 5
+    FiatBalance = UserRole + 6
```

Comment:
The naming is a little off... so we have `FiatValue` which is the asset fiat value + the currency symbol as a string. `FiatBalance` on the other hand is just the fiat value. We might want to rename `fiatValue` to something else in the `Asset` type..

Reply:
Maybe could call `FiatValue` something like `FiatBalanceDisplay` or something similar? That way it can be distinguished as a) different from `FiatBalance` and b) known to be used for displaying the value